bots/dev/pr_comment_parser.py

```
@@ -0,0 +1,213 @@
+    """Fetch all review comments (inline comments) from a GitHub PR using gh CLI.
+
+    Parameters:
+        pr_number (int): The PR number
+        repo (str): Repository in format "owner/repo"
+
+    Returns:
+        List[Dict]: List of review comment dictionaries
+    """
+    # Get review comments (inline comments on code)
+    cmd = ["gh", "api", f"/repos/{repo}/pulls/{pr_number}/comments"]
+    result = subprocess.run(cmd, capture_output=True, text=True, encoding="utf-8", check=True)
```

Ruff (0.13.3)

`subprocess` call: check for execution of untrusted input

(S603)